src/main/java/com/spotify/reaper/resources/view/ClusterStatus.java

```diff
@@ -16,6 +16,8 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.spotify.reaper.core.Cluster;
 
+import jersey.repackaged.com.google.common.collect.Lists;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'jersey.repackaged.com.google.common.collect.Lists'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)